gui/doc/viselements.json

```diff
@@ -0,0 +1,1252 @@
+{
+  "controls": [
+    [
+      "text",
```

Comment:
the construct [[name, description]] does not garantee the unicity. What's the rational behind that choice ?